frontend/src/scenes/experiments/MetricsView/legacy/ChartEmptyState.tsx

```diff
@@ -1,7 +1,7 @@
 import { IconActivity, IconClock } from '@posthog/icons'
 import { LemonTag } from '@posthog/lemon-ui'
 
-import { EXPERIMENT_MIN_EXPOSURES_FOR_RESULTS } from '../constants'
+import { EXPERIMENT_MIN_EXPOSURES_FOR_RESULTS } from '../../constants'
```

Comment:
Nitpick: Using `~/scenes/experiments/constants` rather than two levels down is clearer.